test/sequential/test-net-connect-emfile.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+var net = require('net');
+var common = require('../common');
+var assert = require('assert');
+
+console.log('Test to check the error message format duing EMFILE error');
+
+if (process.platform === 'win32') {
+  console.log('Skipping test, no RLIMIT_NOFILE on Windows.');
+  return;
+}
+
+// Server listens
+net.createServer().listen(common.PORT);
+
+// Connect to server
+for (var i = 0; i < 100000; i++) {
```

Comment:
But that will not reproduce this error because this particular undefined:undefined is added specifically in net module